src/components/ux-subnavitem/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
It's not really necessary to include all the boilerplate for a subcomponent like this, is it? Just the handlebars and the js would suffice?